pages/_app.js

```diff
@@ -6,6 +6,9 @@ export default function App({ Component, pageProps }) {
   const allZukunftsanalyseEntries = placeholderEntries.filter(
     (placeholderEntry) => placeholderEntry.type === "FutureAnalysis"
   );
+  const allVergangenheitsanalyseEntries = placeholderEntries.filter(
```

Comment:
```suggestion
  const allPastAnalysisEntries = placeholderEntries.filter(
```
Also here (see above).